be/src/olap/rowset/rowset.h

```diff
@@ -152,8 +268,13 @@ class Rowset : public std::enable_shared_from_this<Rowset> {
     bool _is_pending;    // rowset is pending iff it's not in visible state
     bool _is_cumulative; // rowset is cumulative iff it's visible and start version < end version
 
-    DorisCallOnce<OLAPStatus> _load_once;
+    SpinLock _lock;
+    Mutex _load_lock;
```

Comment:
give comment for the usage of these two lock

Reply:
done